Test mercury with ucx on leap15

brianjmurrell opened the pull request.

brianjmurrell committed:
DAOS-9919, CORCI-1170, CORCI-1171 ci: Use Artifactory

For Dockerfiles, package building and functional testing.

Fall back to Nexus repos if Artifactory repos fail for any reason and
fall back to on-image repo definitiion of fetching of repos fails for
any reason.

Move bulk of package instaltion in Leap 15 Dockerfile to DNF to avoid
cache coherency issues on repository proxies.

Repo-files-PR: PR-1
Skip-PR-comments: true
Skip-build-el7-gcc: false
Skip-build-leap15-icc: false
Skip-bullseye: false
Skip-func-test-vm-valgrind: false
Skip-func-test-el7: false
Skip-func-test-el8: false
Skip-func-test-leap15: false
Allow-unstable-test: true
Test-tag: pr load_mpi core_files

Signed-off-by: Brian J. Murrell <brian.murrell@intel.com>

brianjmurrell committed:
Test mercury with ucx on leap15

Quick-functional: true
PR-repos: mercury@PR-76
PR-repos-leap15: openucx@PR-1
RPM-test-version: 2.1.100-24.7935.g1c2c6725
Skip-func-test-leap15: false
Skip-func-test-el8: false
Skip-func-test-el7: false

Signed-off-by: Brian J. Murrell <brian.murrell@intel.com>

daosbuild1 commented:
Test stage Functional on CentOS 7 completed with status FAILURE.  https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-8596/1/display/redirect

daosbuild1 commented:
Test stage Functional on EL 8 completed with status FAILURE.  https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-8596/2/display/redirect

brianjmurrell committed:
Verify UCX package versions

Repo-files-PR: PR-1
Quick-functional: true
PR-repos: mercury@PR-76
RPM-test-version: 2.1.100-24.7935.g1c2c6725
Skip-func-test-leap15: false
Skip-func-test-el8: false
Skip-func-test-el7: false

Signed-off-by: Brian J. Murrell <brian.murrell@intel.com>

brianjmurrell committed:
Allow vendor change

Repo-files-PR: PR-1
Quick-functional: true
PR-repos: mercury@PR-76
PR-repos-leap15: openucx@PR-1
RPM-test-version: 2.1.100-24.7935.g1c2c6725
Skip-func-test-leap15: false
Skip-func-test-el8: false
Skip-func-test-el7: false

Signed-off-by: Brian J. Murrell <brian.murrell@intel.com>

brianjmurrell committed:
Update to newer kernel for test

Repo-files-PR: PR-1
Quick-functional: true
PR-repos: mercury@PR-74:19
RPM-test-version: 2.1.100-24.7941.g4b0ec3c2
Skip-func-test-leap15: false
Skip-func-test-el8: false
Skip-func-test-el7: false
Test-tag: load_mpi

Signed-off-by: Brian J. Murrell <brian.murrell@intel.com>

daosbuild1 commented:
Test stage Functional on Leap 15 completed with status FAILURE.  https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-8596/5/display/redirect

daosbuild1 commented:
Test stage Functional on EL 8 completed with status FAILURE.  https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-8596/5/display/redirect

daosbuild1 commented:
Test stage Functional on CentOS 7 completed with status FAILURE.  https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-8596/5/display/redirect
